### PR TITLE
camelCase `eventEmitter` file name

### PR DESCRIPTION
**Pull request by jonathanp**

### CODE REVIEW

**Comment by mateuszluczak** [comment]
Hey thanks for the contribution,

This change makes sense however test specs also need to be updated, because now they will fail.

I've forgotten to enable builds for PR on Circle CI, but now it should work and catch such changes.

**Comment by jonathanp** [comment]
Good catch! Seems to be some problem with the Selenium server now.

**Comment by mateuszluczak** [comment]
Interesting, I need to check it, otherwise your code looks good! I will try to fix the selenium and merge your code. 

Cheers

File: test/unit/decorators/componentSpec.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ class TestComponent {
 describe('Component Decorator', () => {
   it('registers', () => {
     const registry = new Registry();
+    registry.registerComponent('test', TestComponent);
```
Comment (mateuszluczak) [inline test/unit/decorators/componentSpec.js]: Is this change required? Was this test case previously failing? Decorator should handle the registration, and `Registry ` is a singleton

Comment (jonathanp) [reply test/unit/decorators/componentSpec.js]: That test failed when I added the "clears components" test in registrySpec.js. Each unit test should be self contained and shouldn't depend on what happens in a previous test. But you are correct in that now we lost the test that tested if the decorator registers correctly. Hmm, maybe register the component in a `beforeEach` hook instead? That way all the unit tests are self contained, and `registry.registerComponent` wouldn't be imperativly called.

Comment (mateuszluczak) [reply test/unit/decorators/componentSpec.js]: That's true and I agree 100%. But the idea of this test case was to check if `@Component` decorator registers component, so calling explicitly `registerComponent` kinda doesn't test it at all

**Comment by mateuszluczak** [comment]
E2E tests are working on CircleCi

File: test/unit/decorators/componentSpec.js (hunk 1 above)
Comment (jonathanp) [reply test/unit/decorators/componentSpec.js]: Ok I understand, I changed the tests now to re-declare the Component before every test, so it isn't affected by previous tests. What do you think of that solution?

**Comment by mateuszluczak** [comment]
Hey @jonathanp I've seen you've fixed the code, but tests still failed, I think you forgot to merge `master` branch to your pull request - it should help with failing e2e tests.

**Comment by jonathanp** [comment]
Thanks!

**Pull request merged**